builder/src/main/java/cz/xtf/builder/db/MySQL.java

```diff
@@ -1,36 +1,37 @@
 package cz.xtf.builder.db;
 
 import cz.xtf.builder.builders.pod.PersistentVolumeClaim;
+import cz.xtf.core.config.XTFConfig;
 import cz.xtf.core.image.Image;
 
 public class MySQL extends AbstractSQLDatabase {
 
     public MySQL() {
-        super("MYSQL", "/var/lib/mysql/data");
+        super("MYSQL", XTFConfig.get("xtf.mysql.datadir", "/var/lib/mysql/data"));
```

Comment:
Could you move those into the constant like: 
```
private static final String DATA_DIR_PROPERTY = "xtf.mysql.datadir";
private static final String MYSQL_DATA_DIR = XTFConfig.get(DATA_DIR_PROPERTY, "/var/lib/mysql/data");
```
and use MYSQL_DATA_DIR here, please?

Reply:
this way the value would be loaded all the times, not just at object creation, is that ok?

Reply:
> @mnovak1 this way the value would be loaded all the times, not just at object creation, is that ok?

Actually I would resign on this and use my way :-) the problem is that you need to call `XTFConfig.loadConfig()` anytime you want to load updated system properties. So above will not help unless `XTFConfig.loadConfig()` is called before from the test code.

Reply:
@mnovak1 thank you! done

Reply:
@mnovak1 I just verified it works with Red Hat image (see eap-8.x-openshift-4-wildfly-openjdk17/505)